app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,6 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.AUTHENTICATE_ACCOUNTS" />
-    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
```

Comment:
Seems like you accidentally worked on the same branch with your previous PR. These changes seems irrelevant with current PR and should be reverted.